test/lib/mock-dbus.ts

```diff
@@ -0,0 +1,48 @@
+import * as dbus from 'dbus';
+import { DBusError, DBusInterface, busType, DBusConnection } from 'dbus';
+import { createSandbox, SinonSandbox, SinonStub } from 'sinon';
+
+import * as dbusLib from '../../src/lib/dbus';
+
+let sandbox: SinonSandbox;
+
+const dbusLibFunctions = Object.keys(dbusLib).filter((k) => k !== 'DbusError');
+
+const stubs: { [key: string]: SinonStub } = {};
```

Comment:
Element in this object are never cleared, meaning data will leak between tests

Reply:
Oh, I just saw that this object is filled with the list of methods for the stub module

Reply:
I added `$DBUS_STUB.reset()` for every stub acquired via `mockDbus.getStub` but that's not the best, you're right. Maybe I'll remove the method stubs entirely here, and manually create dbus stubs in tests that need them

Reply:
Or maybe `mockDbus.getStub` creates a stub through mockDbus so that the actual dbus does not have to be imported (thus avoiding the import with side effects, if we're not using the memoize implementation)